app/(auth)/signup.tsx

```diff
@@ -6,21 +6,33 @@ import Screen from '@src/components/screen';
 import useSession from '@src/hooks/session';
 import tamaguiConfig from '@src/tamagui';
 import Assets from '@src/theme/assets';
-import { Link } from 'expo-router';
+import { Link, router } from 'expo-router';
 import { useCallback } from 'react';
 import { H5, Image, SizableText, XStack, YStack } from 'tamagui';
 
 function SignupScreen() {
     const { currentUser, isLoggedIn } = useSession();
     const onPressContinueEmail = useCallback(() => {
```

Comment:
This was just a example function, we should do this inside register form which can handle such things.

Reply:
yeah definitely, we will have to implement a email sign in screen later on